### PR TITLE
removed resource properties from score files

### PR DESCRIPTION
**Pull request by sujaya-sys**
#### Description
We are planning to deprecate resource properties in the Score spec: https://github.com/score-spec/spec/issues/61

#### What does this PR do?
It removes resource properties from the example score.yaml files we store in this repo

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New chore (expected functionality to be implemented)

#### Checklist:

- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [x] I've signed off with an email address that matches the commit author.

### CODE REVIEW

**Review by ghen — CHANGES_REQUESTED** [review]
CLI code needs to be updated. Otherwise these examples won't work and that would confuse the users.

Please wait for code changes before approving/merging these changes.

**Review by ghen — COMMENTED** [review]
LGTM.
Would appreciate someone else to review these changes as well.

File: internal/helm/convert_test.go [hunk 1]
```diff
@@ -123,25 +123,12 @@ func TestScoreConvert(t *testing.T) {
...
 					},
 					"app-db": {
-						Properties: map[string]score.ResourcePropertySpec{
-							"host":      {Default: "localhost", Required: true},
-							"port":      {Default: 5432, Required: false},
-							"name":      {Required: true},
-							"user.name": {Required: true, Secret: true},
-							"password":  {Required: true, Secret: true},
-						},
+						Type: "posgres",
```
Comment (chrishumanitec) [inline internal/helm/convert_test.go:128]: should be `postgres` missing a `t`

**Review by chrishumanitec — CHANGES_REQUESTED** [review]
One typeo otherwise looks good.

File: internal/helm/convert_test.go (hunk 1 above)
Comment (ghen) [reply internal/helm/convert_test.go]: Fixed.

**Review by ghen — APPROVED** [review]

**Pull request merged**